### PR TITLE
feat: ✨ themes for bat and nb

### PR DESCRIPTION
**Pull request by tkapias**
I duplicated sublime/falcon.tmTheme to use it for [bat](https://github.com/sharkdp/bat).

I put it together with a config file and instructions for [nb](https://github.com/xwmx/nb) which uses bat.

### CODE REVIEW

File: bat-nb/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## bat
```
Comment (fenetikm) [inline bat-nb/README.md:1]: can you please split this PR into having a directory for each separately? i.e. `bat/` and `nb/` ?

Comment (fenetikm) [inline bat-nb/README.md:1]: can you please split this PR into having a directory for each separately? i.e. bat/ and nb/ ?

**Review by fenetikm — CHANGES_REQUESTED** [review]

**Comment by tkapias** [comment]
I separated them in 2 folders.

I also left a TODO file for a tool related to nb: ACE Editor. I can delete it if it's the wrong place to keep the info.

**Comment by tkapias** [comment]
Closing this as I split this PR, first in PR #88 for bat, and I will add another one for nb.

**Pull request closed without merging**